src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,144 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://pants.readme.io/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0.dev3 (7/14/2020)
+----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Make help-all dump a JSON blob containing all help info. (#10336)
+  `PR #10336 <https://github.com/pantsbuild/pants/pull/10336>`_
+
+* Integrate option value derivation into help. (#10313)
+  `PR #10313 <https://github.com/pantsbuild/pants/pull/10313>`_
+
+* Generate subtargets with file-level precision for dependency inference (#10305)
+  `PR #10305 <https://github.com/pantsbuild/pants/pull/10305>`_
+
+* Add option for specifying environment variable with concurrency slot (#10297)
+  `PR #10297 <https://github.com/pantsbuild/pants/pull/10297>`_
+
+* Add support for specifying a custom coverage config file (.coveragerc) (#10289)
+  `PR #10289 <https://github.com/pantsbuild/pants/pull/10289>`_
+
+* Run code coverage on v2 unit tests. (#9919)
+  `PR #9919 <https://github.com/pantsbuild/pants/pull/9919>`_
+
+Bugfixes
+~~~~~~~~
+
+* Fix generated subtargets not working with `dependees` (#10355)
+  `Issue #10354 <https://github.com/pantsbuild/pants/issues/10354>`_
+  `PR #10355 <https://github.com/pantsbuild/pants/pull/10355>`_
+
+* Ignore dead symlinks on Linux as well as OSX (#10339)
+  `Issue #255) <https://github.com/notify-rs/notify/issues/255)>`_
+  `PR #10339 <https://github.com/pantsbuild/pants/pull/10339>`_
+
+* Fix bug where workunit completion was not reported correctly (#10277)
+  `PR #10277 <https://github.com/pantsbuild/pants/pull/10277>`_
+
+* Fix collision in workunit affecting patches. (#10350)
+  `PR #10350 <https://github.com/pantsbuild/pants/pull/10350>`_
+
+* Revert modernization of virtualenv to fix pants.pex deploy in CI. (#10343)
+  `PR #10343 <https://github.com/pantsbuild/pants/pull/10343>`_
+
+* Fix `run_virtualenv` to target the current `$PY`. (#10331)
+  `PR #10331 <https://github.com/pantsbuild/pants/pull/10331>`_
+
+* Fix awslambda configuration of Pex. (#10323)
+  `PR #10323 <https://github.com/pantsbuild/pants/pull/10323>`_
+
+* Hotfix Bandit breaking from stevedore 3.0 release (#10319)
+  `PR #10319 <https://github.com/pantsbuild/pants/pull/10319>`_
+
+* Remove -q from the release run of reversion. (#10301)
+  `PR #10301 <https://github.com/pantsbuild/pants/pull/10301>`_
+
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Add check for invalid concrete time dates (#10349)
+  `PR #10349 <https://github.com/pantsbuild/pants/pull/10349>`_
+
+* Pass stream workunit context to all callbacks calls instead of just passing it in the last call. (#10340)
+  `PR #10340 <https://github.com/pantsbuild/pants/pull/10340>`_
```

Comment:
this is a bugfix in a public api (stream work unit handler) so this should be under bug fixes.